.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 # These owners will be the default owners for everything in the
 # repo. Unless a later match takes precedence, these owners will be
 # requested for review when someone opens a pull request.
-* @dav3r @felddy
+* @dav3r @felddy @hillaryj @jsf9k @mcdonnnj
```

Comment:
No complaints, but just confirming we're changing the CODEOWNERS intentionally.

Reply:
Yes, I added the rest of us since @felddy doesn't have as much time for reviews these days.